sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -70,8 +70,8 @@ abstract class StringRegexExpression extends BinaryExpression
  * Simple RegEx pattern matching function
  */
 @ExpressionDescription(
-  usage = "str _FUNC_ pattern - Returns true if str matches pattern, " +
-    "null if any arguments are null, false otherwise.",
+  usage = "str _FUNC_ pattern[ escape] - Returns true if str matches `pattern` with `escape`" +
```

Comment:
`[ escape]` -> `[ ESCAPE escape]`?

Reply:
OK. I have thought about whether or not to do this. Under your suggestion, I feel more clear.